jdaviz/configs/imviz/plugins/catalogs/catalogs.py

```diff
@@ -218,9 +229,23 @@ def search(self, error_on_fail=False):
         # markers are added to the viewer based on the table
         viewer.marker = {'color': 'red', 'alpha': 0.8, 'markersize': 5, 'fill': False}
         viewer.add_markers(table=catalog_results, use_skycoord=True, marker_name=self._marker_name)
-
+            
         return skycoord_table
 
+    def plot_selected_points(self,viewer):
```

Comment:
`viewer.add_markers` is expensive because you create a new table entry in data collection and all the relinking happens. If you want to interactively inspect stuff in existing table, maybe we can use the stuff from Markers plugin instead (not to be confused with Astrowidgets markers). @kecnry , since you wrote the Markers plugin, any ideas?

Reply:
I already showed it is possible to circle something, so maybe we can add additional circle overlay to selected data?

https://github.com/spacetelescope/jdaviz/pull/3092

Reply:
If the intention is to "highlight" the currently selected rows when the plugin is opened, then `PluginMark` would be appropriate (that's what is used for all plugin "live-previews").